### ISSUE TITLE
Lack of a11y unit testing.

### ISSUE THREAD

**Issue by harryhmc** [NHSDigital/nhsuk-react-components#7]
PR #6 was raised to solve an a11y issue in one of the components that was not caught by the built-in Storybook tests.

In order to prevent such problems in the future, it would be good to have unit tests covering the accessibility of each component in their respective test files.
